project/depends.scala

```diff
@@ -0,0 +1,22 @@
+import sbt._
+
+object depends {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Object name does not match the regular expression '^[A-Z][A-Za-z0-9]*$'](https://www.codacy.com/app/pathikrit/better-files/file/3776472232/issues/source?bid=2860326&fileBranchId=3676352#l3)